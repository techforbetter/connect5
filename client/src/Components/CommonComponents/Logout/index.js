import React, { Component } from "react";
import styled from "styled-components";

const Btn = styled.button`
   background: none;
   border: none
`;

const I = styled.i`
  font-size: 30px;
  color: var(--logout-button);
`;

const IconDiv = styled.div`
  display: flex;
`;


class Logout extends Component {
  render() {
    const { logout } = this.props;
    return (
      <IconDiv>
        <Btn type="button" onClick={logout}><I className="fas fa-sign-out-alt" /></Btn>
      </IconDiv>
    );
  }
}

export default Logout;
