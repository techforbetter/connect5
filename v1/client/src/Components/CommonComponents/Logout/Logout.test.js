import React from "react";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router-dom";

import Logout from "./index";

test("Logout matches snapshot", () => {
  const context = {};
  const component = renderer.create(
    <StaticRouter context={context}>
      <Logout />
    </StaticRouter>,
  );
  expect(component).toMatchSnapshot();
});
