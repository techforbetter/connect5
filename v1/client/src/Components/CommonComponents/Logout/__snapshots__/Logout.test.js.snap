// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout matches snapshot 1`] = `
<div
  className="sc-htpNat cixthU"
>
  <button
    className="sc-bdVaJa krzXQt"
    type="button"
  >
    <i
      className="fas fa-sign-out-alt sc-bwzfXH klZesC"
    />
  </button>
</div>
`;
